packages/salesforcedx-vscode-core/src/commands/forceOrgOpen.ts

```diff
@@ -6,17 +6,87 @@
  */
 
 import {
+  CliCommandExecutor,
   Command,
   SfdxCommandBuilder
 } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import { ContinueResponse } from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import * as vscode from 'vscode';
+import { channelService } from '../channels';
 import { nls } from '../messages';
+import { telemetryService } from '../telemetry';
+import { getRootWorkspacePath, isSFDXContainerMode } from '../util';
 import {
   EmptyParametersGatherer,
   SfdxCommandlet,
   SfdxCommandletExecutor,
   SfdxWorkspaceChecker
 } from './commands';
 
+type CLIOrgData = {
```

Comment:
Once you rebase you could try using a new interface that's very close to this https://github.com/forcedotcom/salesforcedx-vscode/blob/901e9244e04634447bcc2d886f6cfd0e4b83afa9/packages/salesforcedx-utils-vscode/src/cli/orgCreateResultParser.ts#L8